components/title/multisize/src/index.scss

```diff
@@ -0,0 +1,60 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-multisize-primary: #a81c4f !default;
+$c-title-multisize-secondary: $c-gray !default;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
+
+$fw-title-multisize-pre-title: 500 !default;
+$fw-title-multisize-main-title: 500 !default;
+$fw-title-multisize-post-title: $fw-regular !default;
+
+$fz-title-multisize-pre-title: $fz-h2 !default;
+$fz-title-multisize-main-title: 48px !default;
```

Comment:
Done, I have adapted all font-size and font-weight to the ones defined in theme-basic _typography.